components/brave_vpn/browser/connection/ikev2/win/ras_utils.cc

```diff
@@ -382,6 +385,81 @@ RasOperationResult RemoveEntry(const std::wstring& entry_name) {
   return GetRasSuccessResult();
 }
 
+// I've found you can set this manually via PowerShell using the
+// `Set-VpnConnectionIPsecConfiguration` cmdlet:
+// https://docs.microsoft.com/en-us/powershell/module/vpnclient/set-vpnconnectionipsecconfiguration?view=windowsserver2019-ps
+//
+// I've used the following parameters via PowerShell:
+// >> AuthenticationTransformConstants: GCMAES256
+// >> CipherTransformConstants : GCMAES256
+// >> DHGroup : ECP384
+// >> IntegrityCheckMethod : SHA256
+// >> PfsGroup : None
+// >> EncryptionMethod : GCMAES256
```

Comment:
Hate to make CI start over - but maybe we can remove the comments and just keep this:
```
// `Set-VpnConnectionIPsecConfiguration` cmdlet:
// https://docs.microsoft.com/en-us/powershell/module/vpnclient/set-vpnconnectionipsecconfiguration?view=windowsserver2019-ps
```

Because those params were wrong actually; those ended up changing to the new ones we have (below here)

Reply:
done